server/app.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const path = require('path');
+const db = require('../database/index');
+
+const app = express();
+
+app.use(express.static(path.join(__dirname, '../public')));
+// app.use(bodyParser.json());
```

Comment:
Should be used above line 8 if used. Remove comment.

Reply:
Nice catch, accidentally commented that out. Will fix that and re-commit.